configuration_go/schemas/thanos/cache/redis/redis.go

```diff
@@ -0,0 +1,103 @@
+package redis
+
+import (
+	"time"
+
+	"github.com/thanos-io/thanos/pkg/model"
```

Comment:
We can use the local import now from `configuration_go/schemas/thanos/units/units.go`.

Reply:
done